spec/integration/aws_dhcp_options_spec.rb

```diff
@@ -13,13 +13,13 @@
             netbios_name_servers %w(8.8.8.8 8.8.4.4)
             netbios_node_type    2
           end
-        }.to create_an_aws_dhcp_options('test-dhcp-options', configuration: {
-          domain_name: 'example.com',
-          domain_name_servers: %w(8.8.8.8 8.8.4.4),
-          ntp_servers: %w(8.8.8.8 8.8.4.4),
-          netbios_name_servers: %w(8.8.8.8 8.8.4.4),
-          netbios_node_type: 2
-        }).and be_idempotent
+        }.to create_an_aws_dhcp_options('test-dhcp-options', dhcp_configurations: [
+          {:key=>"domain-name", :values=>[{:value=>"example.com"}]},
```

Comment:
Please use appropriate hash syntax while passing values to dhcp_configurations
eg: use {key: "domain-name", values: [{value: "example.com"}]},